hack/verify-code-patterns.sh

```diff
@@ -16,16 +16,28 @@
 
 set -euo pipefail
 
+# Appropriately set grep
```

Comment:
I think the changes here to our test scripts should be moved to another PR, where we can discuss this independently.

Reply:
Undoing the changes, moving them to a different PR